spectral_cube/spectral_cube.py

```diff
@@ -2498,10 +2498,14 @@ def spectral_interpolate(self, spectral_grid,
             If disabled, a warning will be raised when interpolating onto a
             grid that does not nyquist sample the existing grid.  Disable this
             if you have already appropriately smoothed the data.
+        fill_value : float
+            Value for cube interpolates outside of the original range
+            defined in the data
```

Comment:
I had a hard time parsing this; maybe `Value to use for extrapolation outside of the original image's domain.  The default is to use the nearest pixel in the original image.`  Is that correct, about the default?